<?xml version="1.0"?>
<launch>
    <!-- 
    woosh 모바일 로봇 맵 로드 런치 파일
    
    사용법:
        roslaunch ldj_mobile_robot_map load_map.launch
        roslaunch ldj_mobile_robot_map load_map.launch robot_ip:=169.254.128.2
    -->

    <!-- 로봇 연결 파라미터 -->
    <arg name="robot_ip" default="169.254.128.2" doc="로봇 IP 주소"/>
    <arg name="robot_port" default="5480" doc="로봇 WebSocket 포트"/>
    <arg name="robot_identity" default="map_load_ctrl" doc="SDK 클라이언트 식별자"/>

    <!-- 맵 로드 노드 -->
    <node pkg="ldj_mobile_robot_map" type="ldj_load_map.py" name="ldj_load_map" output="screen">
        <param name="robot_ip" value="$(arg robot_ip)"/>
        <param name="robot_port" value="$(arg robot_port)"/>
        <param name="robot_identity" value="$(arg robot_identity)"/>
    </node>

</launch>

